libminifi/include/utils/ValueParser.h

```diff
@@ -126,6 +126,10 @@ class ValueParser {
     }
   }
 
+  std::string rest() const noexcept {
+    return {str.c_str() + offset};
```

Comment:
I think substring is a bit easier to read:
```
return str.substr(offset);
```

Reply:
done